lib/osf-components/addon/components/draft-registration-card/component.ts

```diff
@@ -61,7 +61,7 @@ export default class DraftRegistrationCard extends Component {
                                     const value = answers.value as RegistrationMetadata;
                                     if (value && property.id in value) {
                                         const propertyValue = value[property.id].value;
-                                        if (propertyValue && propertyValue.length) {
+                                        if (Array.isArray(propertyValue) ? propertyValue.length : propertyValue) {
```

Comment:
If `propertyValue` is an empty string, this used to evaluate falsy, but seems like now it'll evaluate truthy?

Reply:
Empty string is falsy. The `propertyValue.length` was only there for arrays (but not explicitly so because it doesn't hurt for strings). It *does* hurt for booleans, which I didn't realize was an option until a little bit ago.